src/client/entity-editor/identifier-editor/identifier-editor.js

```diff
@@ -77,10 +77,10 @@ const IdentifierEditor = ({
 				</div>
 				<div>
 					{
-						identifiers.map((identifier, rowId) => (
+						identifiers.map((identifier) => (
 							<IdentifierRow
-								index={rowId}
-								key={rowId}
+								index={identifier.id}
```

Comment:
Same as with `AliasRow` above, this modification on `index` is currently breaking the functionnality. Please revert the change to index while keeping `key={identifier.id}`